src/System.Net.WebProxy/tests/WebProxyTest.cs

```diff
@@ -33,7 +33,7 @@ public static IEnumerable<object[]> Ctor_ExpectedPropertyValues_MemberData()
 
         [Theory]
         [MemberData(nameof(Ctor_ExpectedPropertyValues_MemberData))]
-        public static void Ctor_ExpectedPropertyValues(
+        public static void WebProxy_Ctor_ExpectedPropertyValues(
```

Comment:
Same question I asked on #21742, I don't understand why we are now prefixing the class name on the test method name.